packages/protoc-gen-connect-query/src/generateTs.ts

```diff
@@ -74,13 +75,22 @@ const generateServiceFile =
     );
     f.print();
 
-    
+    const unaryFunctionType = (method: DescMethod) => [f.import('UnaryFunctions', '@connectrpc/connect-query'), `<${method.input.name}, ${method.output.name}>`]
```

Comment:
@mckelveygreg This looks good, though I think we can simplify by using the helper type `UnaryFunctionsWithHooks` instead of `UnaryFunctions<...> & UnaryHooks<...>`.

Reply:
😅 Ha, yeah that is much more straight forward!
Thanks!